Dockerfile

```diff
@@ -1,5 +1,5 @@
 # base image
-FROM pelias/libpostal_baseimage
+FROM pelias/baseimage
```

Comment:
I think we want to keep using the `libpostal_baseimage` if we're going to continue to support local server mode.

Reply:
Yeah makes sense, also don't add a huge amount of overhead unless it's loads to memory.

Reply:
reverted via rebase